### PR TITLE
Layout Structuring

### PR DESCRIPTION
**Pull request by JamiesonRoberts**
Updates the layout so that it is persistent across pages as well as gives a framework for the general page layouts.

Resolves #21

### CODE REVIEW

**Review by mwickett — COMMENTED** [review]
Looking good! I had a couple of minor nit comments. Regarding making the top nav sticky - let's check in with Curtis on this one. I was under the impression we weren't going to make it sticky because it takes up space, especially on mobile. 

I can see making it sticky on the main view, and the story view but for the new story form I think we need the screen real estate. Can we make the sticky behaviour a prop to the layout so we can customize?

**Comment by JamiesonRoberts** [comment]
@mwickett I definitely can make sticky a prop, but as a heads up it will mean that when we want to set it to something other than the default we will need to set the .getLayout on the page (you can see on this PR the thanks page sets a different layout).

**Comment by mwickett** [comment]
yeah, that's a great point. Let's get a take from Curtis on the sticky behaviour overall and then go from there.

File: layouts/Blank.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (JamiesonRoberts) [reply layouts/Blank.tsx]: Great catch!

File: pages/thanks.tsx [hunk 2]
```diff
@@ -41,9 +43,9 @@ export default function Thanks() {
         <Button color="white" leftIcon={<TwitterIcon />}>
           <a href="https://twitter.com/MyCOVIDStory_CA">Share @MyCOVIDStory_CA</a>
         </Button>
-        {/* <Button variant="outline" size="sm">
-          <Link href="/">Return to Stories</Link>
-        </Button> */}
+        <Button variant="outline" size="sm">
+          <a href="/">Return to Stories</a>
```
[the remark this answers is not in the document]
Comment (JamiesonRoberts) [reply pages/thanks.tsx]: Great catch!

File: pages/thanks.tsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
-import { useEffect } from 'react'
+import React, { useEffect } from 'react'
```
[the remark this answers is not in the document]
Comment (JamiesonRoberts) [reply pages/thanks.tsx]: Great catch!

**Comment by JamiesonRoberts** [comment]
> yeah, that's a great point. Let's get a take from Curtis on the sticky behaviour overall and then go from there.

@mwickett Curtis gave his feedback in slack, non-sticky by default, sticky on the home page. good call on the prop for it! PR has been updated accordingly!

**Review by mwickett — APPROVED** [review]
🎸

**Pull request merged**